docs/src/developer/style.md

```diff
@@ -83,52 +119,114 @@ Ternary operators provide a way to write clean, concise code.  Use good
 judgement.
 
 Good:
->
-    y = x > 0 ? x : -x
+
+```julia
+y = x > 0 ? x : -x
+```
 
 There are many examples in our codebase that use the form ```<cond> ?
 <statement> : <statement>```.  These can be expressed much more clearly in an
 if/else statement.
 
 ## Unit Tests
+
 All code should be tested.
 
 ## Whitespace
+
 * If many function arguments cause the line length to be exceeded, put one
 argument per line. In some cases it may make sense to pair some variables on
 the same line.
->
-    function foo(var1::String,
-                 var2::String,
-                 var3::String,
-                 var4::String,
-                 var5::String,
-                 var6::String)
-
-* Do not surround equal signs with spaces when passing keyword args to a
+
+Good:
+
+```julia
+function foo(
+    var1::String,
+    var2::String,
+    var3::String,
+    var6::T,
+) where T <: Number
+    println("hello world")
+end
+```
+
+Bad:
+
+function foo(var1::String,
+             var2::String,
+             var3::String,
+             var6::T) where T <: Number
+    println("hello world")
+end
+
+* Surround equal signs with spaces when passing keyword args to a
 function or defining default values in function declarations.
+
+Good:
+
+```julia
+bar(;var1 = 1, var2 = 2, var3 = 3) = @show var1, var2, var3
+bar(var1 = 1, var2 = 2, var3 = 3)
+```
+
+Bad:
+
+```julia
+bar(;var1=1, var2=2, var3=3) = @show var1, var2, var3
+bar(var1=1, var2=2, var3=3)
+```
+
+* Prefer elements in an array on separate lines. Follow opening square bracket with a new line and use closing square bracket on a separate new line.
+
+Good:
+
+```julia
+nodes = [
+    Node(1),
+    Node(2),
+    Node(3),
+    Node(4),
+    Node(5),
+];
+```
+
+Bad:
+
+```julia
+nodes = [Node(1), Node(2), Node(3), Node(4), Node(5)];
```

Comment:
Isn't this OK if it fits on one line?

Reply:
We should be more flexible about this, it can lead to unnecessary whitespace tabbing.

Reply:
I added text regarding this in https://github.com/NREL/PowerSystems.jl/pull/317/commits/e78c96eeb066bd5c291e2f455c079ebe66669d37.